ztrouy/readers-digest-api
# Initial Seed Data

### Purpose
To provide initial seed data for Books, Categories, Book Categories, and Reviews

### Files Changed
- Created file, and added initial seed data in new `books.json`, `categories.json`, `book_categories.json`, `reviews.json`, `users.json`, and `tokens.json` fixtures
- Added commands for loading seed data for Books, Categories, Book Categories, and Reviews in `seed_database.sh`
- Added rule to ignore Migrations in `.gitignore`
- Removed original Migration `0001_initial.py`

### To Test
Fetch, setup, and run `seed_database.sh`, then confirm the following
- Table `digestapi_book` has 10 entries
- Table `digestapi_category` has 12 entries
- Table `digestapi_bookcategory` has 26 entries
- Table `digestapi_review` has 9 entries
- Table `auth_user` has 6 entries
- Table `authtoken_token` has 6 entries
